docs/roadmap.rst

```diff
@@ -0,0 +1,88 @@
+Roadmap
+=======
+
+Process
+-------
+
+Read the Docs has adopted the following workflow with regards to how we
+prioritize our development efforts and where the core development team focuses
+its time.
+
+Triaging issues
+~~~~~~~~~~~~~~~
+
+Much of this is already covered in our guide on :doc:`contribute`, however to
+summarize the important pieces:
+
+* New issues coming in will be triaged, but won't yet be considered part of our
+  roadmap.
+* If the issue is a valid bug, it will be assigned the ``Accepted`` label and
+  will be prioritized, likely on an upcoming point release.
+* If the issues is a feature or improvement, the issue might go through a design
+  decision phase before being accepted and assigned to a milestone. This is a
+  good time to discuss how to address the problem technically. Skipping this
+  phase might result in your PR being blocked, sent back to design decision, or
+  perhaps even discarded. It's best to be active here before submitting a PR.
+* The core team will only work on accepted issues, and will only give PR review
+  priority to accepted issues.
+
+Milestones
+~~~~~~~~~~
+
+We maintain two types of milestones: point release milestones for our upcoming
+releases, and group milestones, for blocks of work that have priority in the
+future.
+
+Generally there are 2 or 3 point release milestones lined up. These point
+releases dictate the issues that core team has discussed as priority already.
+Core team should not focus on issues outside these milestones as that implies
+either the issue was not discussed as a priority, or the issue isn't a priority.
+
+Point release milestones try to remain static, but can shift upwards on a
+release that included an unexpected feature addition. Milestones are normally
+realistic and stick to semver, which means that only one feature addition issue
```

Comment:
Might be worth explaining what semver is, or pointing to a description using a link. Same for 'point release'.